src/lib/datastore.jl

```diff
@@ -52,9 +52,9 @@ function cached_fallback(filename, generate)
     confighash = hash(config) # make specific to configs
     try
         if isfile(datapath("$filename$suffix-$confighash.jld"))
-            return deserialize(open(cachepath("$filename$suffix-$confighash.jld")))
+            return deserialize(open(datapath("$filename$suffix-$confighash.jld")))
```

Comment:
maybe better to revert these changes and change line 54 to cachepath

Reply:
Agreed.  And I think you can simplify this whole section, since a lot of the logic is now in cache path.